### ISSUE TITLE
docs: Improve Angular 5 tutorial (remove obsolete code)

### ISSUE THREAD

**Issue by mwalter** [angular/angular#25320]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior
There is a binding annotation for property 'hero' in the hero-detail.component.ts in the Angular 5 tutorial about routes (https://v5.angular.io/tutorial/toh-pt5).

## Expected behavior
The binding of the 'hero' property is no longer needed because a hero is now fetched from the hero service by id. The binding annotation in hero-detail.component.ts is obsolet.
There is no @Inject annotation for property 'hero' anymore.
Maybe you should add some text explaining...

## What is the motivation / use case for changing the behavior?

Make the tutorial better. :-)

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by brandonroberts** [comment]
Thanks @mwalter! V5 docs are no longer receiving updates, but that issue is being fixed in the latest version of the docs once this PR lands https://github.com/angular/angular/pull/22151

Closing as a duplicate of https://github.com/angular/angular/issues/21908
